starlette/datastructures.py

```diff
@@ -422,8 +422,11 @@ class UploadFile:
     An uploaded file included as part of the request data.
     """
 
-    def __init__(self, filename: str, file: typing.IO = None) -> None:
+    def __init__(
+        self, filename: str, content_type: str, file: typing.IO = None
```

Comment:
Let's have `content_type` as the last parameter here, and have it default to `""`.